source/includes/_rendering.md

```diff
@@ -178,4 +177,94 @@ function myRenderOverlay() {
+You can also change the text color, whether or not to render the shadow, change the scale, and whether or not
+the size increases from distance. All of these examples below would work also.
+
+```javascript
+Tessellator.drawString("Blue text!", 100, 100, 100, Renderer.BLUE);
+Tessellator.drawString("Green text with shadow!", 100, 100, 100, Renderer.GREEN, true);
+Tessellator.drawString("Bigger red text!", 100, 100, 100, Renderer.RED, false, 2);
+Tessellator.drawString("Hello there!", 100, 100, 100, Renderer.DARK_PURPLE, true, 2, false);
+```
+
+## Drawing Textures
+
+> You can also draw specific textures like so
+
+```javascript
+register("renderWorld", myWorldRender);
+
+var img = new Image("kirbybit.png", "https://www.chattriggers.com/assets/images/kerbybit.png");
+
+function myWorldRender() {
+  Tessellator.bindTexture(img);
+  Tessellator.begin()
+    .translate(Player.getX(), Player.getY(), Player.getZ())
+    .pos(-0.5, 0.5, -0.5).tex(0, 0)
+    .pos(-0.5, 0.5, 0.5).tex(0, 1)
+    .pos(0.5, 0.5, 0.5).tex(1, 1)
+    .pos(0.5, 0.5, -0.5).tex(1, 0)
+    .draw();
+}
+```
+
+This will draw Kirbybit's skin texture on the player's feet position. This works by first binding the texture
+of the image to the `Tessellator` object. Then, it begins the instructions on what to do,
+by moving the center of the image to the player's position. Then, it has the 4 positions
+for each vertex, with the texture starting in the top left (0, 0) and working its way counter-clockwise.
+Then it's ended with `draw()` to finalize the texture.
+
+### Partial Ticks
+
+Partial ticks are a fractional value representing the amount of time that’s passed between the last full tick and now. This is
+useful in rendering, as the rest of the game runs on the tick system, but rendering uses frames, a much more precise operation.
+We use this because otherwise the animation would be jittery because there are fewer ticks per second than frames per second.
+
+If we wanted to make the example from above not be jittery, we would use partial ticks.
+
+```javascript
+function myWorldRender(partialTicks) {
+  const lastX = new Entity(Player.getPlayer()).getLastX();
+  const lastY = new Entity(Player.getPlayer()).getLastY();
+  const lastZ = new Entity(Player.getPlayer()).getLastZ();
+
+  const currentX = Player.getX();
+  const currentY = Player.getY();
+  const currentZ = Player.getZ();
+
+  Tessellator.bindTexture(img);
+  Tessellator.begin()
+    .translate(
+      lastX + (currentX - lastX) * partialTicks,
+      lastY + (currentY - lastY) * partialTicks,
+      lastZ + (currentZ - lastZ) * partialTicks
+    )
+    .pos(-0.5, 0.5, -0.5).tex(0, 0)
+    .pos(-0.5, 0.5, 0.5).tex(0, 1)
+    .pos(0.5, 0.5, 0.5).tex(1, 1)
+    .pos(0.5, 0.5, -0.5).tex(1, 0)
+    .draw();
+}
+```
+
+The key difference in this that the change in position is multiplied by the change in time every frame. This makes
+the view smooth since it updates each frame instead of every tick.
```

Comment:
```suggestion
The key difference here is that the change in position is multiplied by the change in time every frame. 
This makes the drawing smooth since it updates every frame instead of every tick.
```